plugins/org.eclipse.n4js.flowgraphs/src/org/eclipse/n4js/flowgraphs/model/Node.java

```diff
@@ -0,0 +1,170 @@
+/**
+ * Copyright (c) 2017 Marcus Mews.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Marcus Mews - Initial API and implementation
+ */
+package org.eclipse.n4js.flowgraphs.model;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import org.eclipse.n4js.flowgraphs.ControlFlowType;
+import org.eclipse.n4js.flowgraphs.factories.CFEMapper;
+import org.eclipse.n4js.n4JS.ControlFlowElement;
+
+/** Represents a part of a {@link ControlFlowElement}. */
+abstract public class Node implements ControlFlowable {
```

Comment:
I thought about renaming the `Node` to `InternalNode`, but then I was reluctant because the actual control flow graph consists of these nodes.
I extended the JavaDoc though.